packages/docker/Dockerfile.vite

```
@@ -0,0 +1,65 @@
+ENV DEPLOYMENT_ENV $DEPLOYMENT_ENV
+
+ARG SENTRY_AUTH_TOKEN
+ARG GIT_COMMIT_HASH
+
+WORKDIR /app
+
+FROM base as prepare
+
+COPY package*.json /app
+COPY /$WORKSPACE/package.json /app/$WORKSPACE/package.json
+COPY --parents /packages/*/package.json /app
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)